setup.py

```diff
@@ -57,40 +57,37 @@
         "ecdsa",
         "bech32",
         "requests",
-        "google-api-python-client",
         "protobuf>=3.19.4,<4",
-        "python-mbedtls==2.2.0",
-        "grpcio",
-        "click",
-        "types-certifi",
+        "grpcio==1.47.0",
         "bip-utils",
         "blspy",
+        "google-api-python-client",
     ],
     extras_require={
         "dev": [
             "check-manifest",
-            "tox==3.24.1",
+            "tox==3.25.1",
             "flake8==3.9.2",
-            "black==22.3",
-            "mypy==0.910",
+            "black==22.6",
+            "mypy==0.971",
             "mkdocs==1.3",
+            "markdown==3.3.4",
```

Comment:
Do we need "markdown" specifically added? I think it is a dub-dependency of material-for-mkdocs.

Reply:
done